woocommerce/wc-admin
# Add free extensions data store

Adds a selector to the onboarding store for storing fetched free extensions

### Detailed test instructions:

1. Navigate to the marketing task
2. Make sure extensions are fetched and displayed correctly
